conformance/tests/generics_syntax_declarations.py

```diff
@@ -0,0 +1,82 @@
+"""
+Validates the type parameter syntax introduced in PEP 695.
+"""
+
+# Specification: https://peps.python.org/pep-0695/#type-parameter-declarations
+
+
+# This generic class is parameterized by a TypeVar T, a
+# TypeVarTuple Ts, and a ParamSpec P.
+from typing import Generic, ParamSpec, Protocol, TypeVar, TypeVarTuple, assert_type
+
+
+class ClassA[T, *Ts, **P]:
+    assert_type(T, TypeVar)
+    assert_type(Ts, TypeVarTuple)
+    assert_type(P, ParamSpec)
+
+
+class ClassB[T](Generic[T]):  # Runtime error
+    ...
+
+
+class ClassC[S, T](Protocol[S, T]):  # Type error
```

Comment:
we lost the `class ClassA[S, T](Protocol)  # OK` example

Reply:
Added the missing test.